controller/controller.go

```
@@ -98,6 +107,28 @@ func Filter(gvk schema.GroupVersionKind) func(obj interface{}) bool {
 	}
 }
 
+// FilterGroupKind makes it simple to create FilterFunc's for use with
+// cache.FilteringResourceEventHandler that filter based on the
+// schema.GroupKind of the controlling resources.
+func FilterGroupKind(gk schema.GroupKind) func(obj interface{}) bool {
+	return func(obj interface{}) bool {
+		if object, ok := obj.(metav1.Object); ok {
+			owner := metav1.GetControllerOf(object)
+			if owner == nil {
+				return false
+			}
+
+			ownerGV, err := schema.ParseGroupVersion(owner.APIVersion)
```

since you're ignoring error, you can just `return err==nil && ownerGV.Group...`